src/apis/platform/api.ts

```diff
@@ -427,6 +425,18 @@ class PlatformAPI extends JRPCAPI{
         });
     }
 
+    /**
+     * Get all the subnets that exist.
+     * 
+     * @returns Promise for an array of objects containing fields "id", "controlKeys", and "threshold".
+     */
+    getSubnets = async ():Promise<Array<object>> => {
+        let params = {}
+        return this.callMethod("platform.getSubnets", params).then((response:RequestResponseData) => {
+            return response.data["result"];
```

Comment:
This line isnt correct I think. It should return 

```js
return response.data["result"]["subnets"];
```

Which is an array of objects.